src/main/msp/msp.c

```diff
@@ -643,8 +641,15 @@ static bool mspCommonProcessOutCommand(int16_t cmdMSP, sbuf_t *dst, mspPostProce
         sbufWriteU8(dst, FC_VERSION_PATCH_LEVEL);
         break;
 
-    case MSP_BOARD_INFO:
-    {
+    case MSP2_MCU_TYPE_INFO: {
+        sbufWriteU8(dst, getMcuTypeId());
+        const char *name = getMcuTypeName();
+        sbufWriteU8(dst, strlen(name));
+        sbufWriteString(dst, name);
```

Comment:
(we should implement sbufWritePString (pascal-string), it is used in a lot of places)